metadata.yaml

```diff
@@ -28,3 +28,7 @@ requires:
     interface: http
   saml:
     interface: saml
+  smtp:
+    interface: saml
+  smtp-legacy:
+    interface: saml
```

Comment:
I'm adhering to the spec https://docs.google.com/document/d/13K6TlXI2b_Hg0-0_b0LeMhPJ3WVdfJtFPsImVRgiFNc

Reply:
I think in the spec, the interface name is `smtp` instead of `saml`?

```
provides:
  smtp:
    interface: smtp
  smtp_legacy:
    interface: smtp
```

Reply:
Sorry I just found there are some typos in the comment last week.

Reply:
Indeed